ethereumj-core/src/main/java/org/ethereum/db/RepositoryRoot.java

```diff
@@ -129,7 +129,9 @@ public synchronized void flush() {
 
     @Override
     public Repository getSnapshotTo(byte[] root) {
-        return new RepositoryRoot(stateDS, root);
+        RepositoryRoot repo = new RepositoryRoot(stateDS, root);
+        repo.config = this.config;
```

Comment:
pls, move this argument to the constructor

Reply:
Kk